wiki/Contests/oMWC/2/en.md

```diff
@@ -0,0 +1,158 @@
+14. **Only one team per country will be allowed** - While some countries are more active than others on osu! and in mapping, to keep everything fair between all of the participants of this contest, only one team will be made per country.
+15. **Keep the organizers in touch with team updates past the preparation phase** - If a contestant has to drop from the contest for any reasons, please let the staff know about it as soon as possible. No replacements will be allowed and no rewards will be attributed to the user if the team gets into the top 3.
+
+### Questions and answers
+
+1. **What is this contest?**
+
+   The osu! Beatmapping World Championship is a standard mode contest where several countries compete against each other to show off their country’s best mapping ability. Each team member will be handpicked by a Team Captain, composed of 3 to 7 members, including the Team Captain.
+
+2. **How does this contest work?**
+
+   The contest will have 2 stages with a total of 5 rounds. The first round is a qualifier round where all eligible country teams will compete against each other in a typical mapping contest with one song to map. The top 16 placing teams will move onto the next stage of the contest. After the qualifier round, the remaining stages will follow a head to head bracket format, also called “Clash Rounds”. For each matchup between a pair of two teams, each team will be allowed to ban one genre among the three available which will lead them to map the remaining song. Once the mapping phase is over, the judges will be tasked to vote for the best entries out of the two teams, and so until the final.  
+
+3. **What will be the songs?**
+
+   The song genres will be revealed for all phases prior the qualifiers, while the song names for each stage will be announced as soon as the stage starts.
+
+4. **What is a “Team Captain”?**
+
+   The Team Captains, also called Captains are chosen by the mappers of their country through a vote on the website. They will be the ones in charge to communicate with the organizers and their team in order to let us know about team-related things. The requirements to become Team Captain is to have at least one ranked guest difficulty, in order to show your mapping experience.
+
+5. **What if my country doesn’t have a Team Captain?**
+
+   You’re free to apply as one through the website if you meet the criterias! Mappers of your country will vote for the people they want to have as their country’s Team Captain. If nobody applied to be a Team Captain for your country once the deadline is over, contest staff will pick one for the team. The country you will be allowed to participate with will be based on your profile's flag. No country merge will be allowed to avoid management complications.
```

Comment:
```suggestion
   You’re free to apply to become one through the website if you meet the criteria! Mappers of your country will vote for the people they want to have as their country’s Team Captain. If nobody applied to be a Team Captain for your country once the deadline is over, contest staff will pick one for the team. The country you will be allowed to participate with will be based on your profile's flag. No country mergers will be allowed to avoid management complications.
```